src/plotting_tools.py

```diff
@@ -413,6 +413,7 @@ def set_mode(self, mode):
         cut_button.set_visible(True)
         highlight.set_visible(True)
         highlight.set_active(True)
+        self.dummy_toolbar.mode = "select/cut"
```

Comment:
setting the mode this way is a bad idea.
matplotlib's NavigationToolbar2 uses an enum to set its mode. These values can only be `_Mode.PAN` ("pan/zoom"), `_Mode.ZOOM` ("zoom rect") and [_Mode.NONE](url) (""). This enum allows for string comparison e. g. `if (mode == "pan/zoom"`, but setting the mode to any string can cause issues with some internal functions.
Rebasing on https://github.com/SjoerdB93/Graphs/pull/67 and using self._mode for all checks would be a better solution.